### PR TITLE
New README

### PR DESCRIPTION
**Pull request by tlienart**
Preview: https://github.com/tlienart/MLJ.jl/blob/master/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +51,41 @@ MLJ is released under the MIT licensed and sponsored by the [Alan Turing Institu
...
+For instance, if you want to use a **Decision Tree Classifier**, you need to have [DecisionTree.jl](https://github.com/bensadeghi/DecisionTree.jl) installed:
+
+```julia
+julia> Pkg.add("DecisionTree");
+julia> using MLJ;
+julia> @load DecisionTreeClassifier
+```
+
+For a list of models and their packages see the [table below](#models-available), we recommend you start with models marked as coming from _mature_ packages such as _DecisionTree_, _ScikitLearn_ or _XGBoost_.
+
```
Comment (ablaom) [inline README.md]: Replace with
"For a list of models and their packages  set the [table below](#models-available) or run `using MLJ; models()`. We recommend ..."

File: README.md [hunk 2]
```diff
@@ -173,24 +195,11 @@ The table below indicates the models that are accessible at present along with a
...
-- Franz Kiraly
-- Zac Nugent
-- Yiannis Simillides
+*Active maintainers*: A. Blaom, T. Lienart
+
+*Active collaborators*: D. Arenas, D. Buchaca, J. Hoffimann, S. Okon, J. Samaroo, S. Vollmer
+
+*Past collaborators*: D. Aluthge, E. Barp, G. Bohner, M. K. Borregaard, V. Churavy, H. Devereux, M. Giordano, M. Innes, F. Kiraly, M. Nook, Z. Nugent, P. Oleśkiewicz, A. Shridar, Y. Simillides, A. Sengupta, A. Stechemesser.
 
 #### License
```
Comment (ablaom) [inline README.md]: I have given this some thought, and I think it is important to acknowledge Sebastian and Franz in the design. Can we please add:

*Core design*: A. Blaom, F. Kiraly, S. Vollmer

**Comment by ablaom** [comment]
Thanks again for this @tlienart 

The links at the top for Cheatsheet and Citing MLJ need fixing.

File: README.md [hunk 3]
```diff
@@ -1,159 +1,184 @@
...
+
+---
+
+### The MLJ Universe
+
+The MLJ universe is made out of several repositories some of which can be used independently of MLJ (indicated with a ⟂ symbol):
+
+* (⟂) [MLJBase.jl](https://github.com/alan-turing-institute/MLJBase.jl) offers essential tools to load and interpret data, describe ML models and use metrics; it is the repository you should interface with if you wish to make your package accessible via MLJ,
+* [MLJ.jl](https://github.com/alan-turing-institute/MLJ.jl) offers tools to compose, tune and evaluate models,
+* [MLJModels.jl](https://github.com/alan-turing-institute/MLJ.jl) contains interfaces to a number of important model-providing packages such as,  [DecisionTree.jl](https://github.com/bensadeghi/DecisionTree.jl), [ScikitLearn.jl](https://github.com/bensadeghi/DecisionTree.jl) or [XGBoost.jl](https://github.com/dmlc/XGBoost.jl) as well as a few built-in transformations (one hot encoding, standardisation, ...), it also hosts the *model registry* which keeps track of all models accessible via MLJ,
```
Comment (ablaom) [inline README.md]: Link to wrong repo

File: README.md [hunk 4]
```diff
@@ -1,159 +1,184 @@
...
+* *high*: indicates the package is very mature and functionalities are expected to have been fairly optimised and tested.
+
+| Package | Models | Maturity | Note
+| ------- | ------ | -------- | ----
+[Clustering.jl] | KMeans, KMedoids | high | [note]
+[DecisionTree.jl] | DecisionTreeClassifier, DecisionTreeRegressor | high | [note]
+[GaussianProcesses.jl] | GPClassifier | medium | [note]
+[GLM.jl] | LinearRegressor, LinearBinaryClassifier, LinearCountRegressor | medium | [note]
+[LIBSVM.jl] | LinearSVC, SVC, NuSVC, NuSVR, EpsilonSVR, OneClassSVM | high | also via ScikitLearn.jl
+[MLJModels.jl] (builtins) | StaticTransformer, FeatureSelector, FillImputer, UnivariateStandardizer, Standardizer, UnivariateBoxCoxTransformer, OneHotEncoder, ConstantRegressor, ConstantClassifier, (KNNRegressor) | medium |
```
Comment (ablaom) [inline README.md]: Why is KNNRegressor here? Is this a reference to the obsolete version? I think we can just drop this.

File: README.md [hunk 5]
```diff
@@ -1,159 +1,184 @@
...
+[XGBoost.jl] | XGBoostRegressor, XGBoostClassifier, XGBoostCount | high |
+
+[note]: some models are missing, your help is welcome to complete the interface. Get in touch with Thibaut Lienart on Slack if you want to help.
```
Comment (ablaom) [inline README.md]: Finicky request: replace "[note]:" with "*Note*:"

**Comment by ablaom** [comment]
Please remove "GaussianProcesses" from the package list. The implementation there is wrong and the package has been removed some time ago from the registry.  

Also, I would demote "NaiveBayes" maturity to "low". It is old and not well-maintained, has a suspected bug, and our implementation does not support sparsity. (The package itself does, but in a substandard way using dictionaries.)

**Comment by ablaom** [comment]
Also, I though in the "help wanted" section we could add something about improving sparse data support and support for natural language processing. What do think?

If you are happy with my comments, no need for me to revisit. Merge when ready.

File: README.md (hunk 5 above)
Comment (tlienart) [reply README.md]: yes so in proper markdown the `note` should have appeared as a link, it seems Github flavoured one doesn't resolve it... fixed as per your comment

**Comment by tlienart** [comment]
* cheatsheet, citing link fixed
* added `models()`
* added `core design`
* fixed link to mljmodels, sklearn
* removed knnregressor from builtins
* added a note about sparsity  and NLP in the help section.

Will double check then merge

**Pull request merged**